staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_test.go

```diff
@@ -333,10 +371,38 @@ func TestWaitUntilFreshAndList(t *testing.T) {
 	if len(list) != 2 {
 		t.Errorf("unexpected list returned: %#v", list)
 	}
+
+	// list with spec.nodeName index.
+	matchValues = []storage.MatchValue{
+		{IndexName: "not-exist-label", Value: "whatever"},
```

Comment:
The tests serve as examples so if we can make the index names realistic, that'd be great.